### PR TITLE
try to fix failed migration events_analytics_platform, 0001_spans

### PR DESCRIPTION
**Pull request by shcherbak**
fix for syntax error in migration :
Codec DoubleDelta does not accept any arguments.

https://github.com/getsentry/snuba/issues/6355

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: snuba/snuba_migrations/events_analytics_platform/0001_spans.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
     Column("name", String(modifiers=Modifiers(codecs=["ZSTD(1)"]))),
     Column("sampling_factor", Float(64, modifiers=Modifiers(codecs=["ZSTD(1)"]))),
     Column("sampling_weight", Float(64, modifiers=Modifiers(codecs=["ZSTD(1)"]))),
-    Column("sign", Int(8, modifiers=Modifiers(codecs=["DoubleDelta(1)"]))),
+    Column("sign", Int(8, modifiers=Modifiers(codecs=["DoubleDelta", "ZSTD(1)"])),
```
Comment (phacops) [inline snuba/snuba_migrations/events_analytics_platform/0001_spans.py:55]: We don't want the `ZSTD(1)` codec for the `sign` column, we can just remove the `(1)`.

**Review by phacops — CHANGES_REQUESTED** [review]

File: snuba/snuba_migrations/events_analytics_platform/0001_spans.py (hunk 1 above)
Comment (shcherbak) [reply snuba/snuba_migrations/events_analytics_platform/0001_spans.py]: done

**Review by phacops — APPROVED** [review]

**Comment by shcherbak** [comment]
@colin-sentry @phacops I need help to fix pipeline to complete this PR
```
 -- backward migration events_analytics_platform : 0001_spans
Local op: DROP TABLE IF EXISTS eap_spans_local;
Distributed op: DROP TABLE IF EXISTS eap_spans_dist;
-- end backward migration events_analytics_platform : 0001_spans
/home/runner/work/_actions/getsentry/action-migrations/v1.0.8/dist/index.js:8451
        const error = new requestError.RequestError(message, status, {
                      ^

RequestError [HttpError]: Resource not accessible by integration
    at /home/runner/work/_actions/getsentry/action-migrations/v1.0.8/dist/index.js:8451:23
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  status: '403',
  headers: {
  ```
Can You please help with PR?

**Pull request merged**